src/styles.scss

```diff
@@ -1,122 +1,230 @@
 }
 
 .material-icons {
-    font-family: 'Material Icons';
-    font-weight: normal;
-    font-style: normal;
-    font-size: 24px;
-    line-height: 1;
-    letter-spacing: normal;
-    text-transform: none;
-    display: inline-block;
-    white-space: nowrap;
-    word-wrap: normal;
-    direction: ltr;
-    -webkit-font-feature-settings: 'liga';
-    -webkit-font-smoothing: antialiased;
+  font-family: "Material Icons";
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
+  // -webkit-font-feature-settings: 'liga';
+  -webkit-font-smoothing: antialiased;
 }
 
-
 body {
-    font-family: $regular-font
+  font-family: $regular-font;
 }
 
 ._flex-box {
-    display: flex;
-    align-items: center
+  display: flex;
+  align-items: center;
 }
-._responsiveTabel{
-    display: none;
-    width: 100%;
+._responsiveTabel {
+  display: none;
+  width: 100%;
 }
 ._flex-direction-column {
-    flex-direction: column;
+  flex-direction: column;
 }
 
 ._justify-content-center {
-    justify-content: center;
+  justify-content: center;
 }
 
 ._justify-content-space-between {
-    justify-content: space-between;
+  justify-content: space-between;
 }
 
 ._flex-wrap {
-    flex-wrap: wrap !important;
+  flex-wrap: wrap !important;
 }
 
 ._full-width {
-    width: 100%;
+  width: 100%;
 }
-._margin-top{
-    margin-top: 20px !important;
+._margin-top {
+  margin-top: 20px !important;
 }
-._small-margin-top{
-    margin-top: 10px !important;
+._small-margin-top {
+  margin-top: 10px !important;
 }
-._small-margin-bottom{
-    margin-bottom: 10px !important;
+._small-margin-bottom {
+  margin-bottom: 10px !important;
 }
-._margin-bottom{
-    margin-bottom: 20px !important;
+._margin-bottom {
+  margin-bottom: 20px !important;
 }
 ._cursor-pointer {
-    cursor: pointer;
+  cursor: pointer;
 }
-.imageText{
-    color: #5e5f62 !important;
-    font-size: 18px;
-    font-weight: 800;
+.imageText {
+  color: #5e5f62 !important;
+  font-size: 18px;
+  font-weight: 800;
 }
 
 ._green-btn {
-    background: $green-color;
-    color: $white-color;
+  background: $green-color;
+  color: $white-color;
 }
 ._smallScreenText {
-    font-size: 18px;
+  font-size: 18px;
 }
 ._primary-btn {
-    background: $primary-color;
-    color: $white-color;
+  background: $primary-color;
+  color: $white-color;
 }
 span:focus {
-    outline: none;
-    border: 0;
- }
-._helveticaFont{
-    font-family: Helvetica !important;
-}
\ No newline at end of file
+  outline: none;
+  border: 0;
+}
+._helveticaFont {
+  font-family: Helvetica !important;
+}
+
+.mat-icon {
+  background-repeat: no-repeat;
+  display: inline-block;
+  color: $primary-color;
+  height: 20px;
+  width: 24px;
+  font-size: 30px;
+}
+
+.mat-drawer-side {
+  border: none !important;
+}
+
+.dashboard-content {
```

Comment:
all classes should have _ in front for global custom class

Reply:
changed